### ISSUE TITLE
Extension Sync Wont work

### ISSUE THREAD

**Issue by Ateeq72** [shanalikhan/code-settings-sync#1369]
Here is my gist https://gist.github.com/Ateeq72/4d5f7793aec1eb8a50e5f5ffad8927b9

No extension present in extensions.json would get installed on sync

I just get this.

![image](https://user-images.githubusercontent.com/1732622/180797693-6d20d40a-3d59-46c4-9f80-1aacd3090e39.png)

You can see the extension 'Git Graph' is present in the extensions.json and is not installed here.

Not just this. When I try to install/sync my extensions in a fresh VSCode install that wont work either.

![image](https://user-images.githubusercontent.com/1732622/180798049-8caf27ac-c243-4fa4-b005-800f82e3e751.png)

**Comment by YuHuaOu** [comment]
The same problem

**Comment by Cooodddaaa** [comment]
I have the same problem

**Comment by crane168** [comment]
> I have the same problem

yes,me too,The prompt is complete, but the extension is not actually downloaded
![image](https://user-images.githubusercontent.com/26795319/181792529-21951e38-4635-4cc4-8214-9889a95c85b0.png)

**Comment by Cooodddaaa** [comment]
#1341 As a workaround for now

**Comment by Ateeq72** [comment]
Ok thanks.. 👍

**Comment by tryoasnafi** [comment]
I got the same problem, it shows my extension upload was successful, but my gist is empty.
